src/org/jrubyparser/lexer/Lexer.java

```diff
@@ -545,7 +545,7 @@ public void reset() {
         resetStacks();
         lex_strterm = null;
         commandStart = true;
-        if (parserSupport != null) isTwoZero = parserSupport.getConfiguration().getVersion().is2_0();
+        if (parserSupport != null) isTwoZero = parserSupport.getConfiguration().getVersion().is2_3();
```

Comment:
This seems wrong..I wonder if we need two booleans here is there is something 2.0 or 2.3+?